.github/workflows/github-actions.yml

```diff
@@ -41,6 +43,11 @@ jobs:
         env:
           SSH_AUTH_SOCK: /tmp/ssh_agent.sock
         run: git submodule update --init --recursive
-      
+
       - name: Test
-        run: ./gradlew :hmkit-fleet:build
\ No newline at end of file
+        run: ./gradlew :hmkit-fleet:build
+
+      - name: Upload codecov report
+        uses: codecov/codecov-action@v3
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
As this is a public repo, I don't think we need to add the token here.

Reply:
Yes, works without the token